### PR TITLE
feat(vsc): UI change for GA features

### PR DESCRIPTION
**Pull request by tecton**
1. Add configuration to enable GA features.
2. Use "Add features" to add capabilities and resources.

E2E TEST: N/A

WIT: https://msazure.visualstudio.com/Microsoft%20Teams%20Extensibility/_workitems/edit/14100028

### CODE REVIEW

**Review by jayzhang — APPROVED** [review]

File: packages/fx-core/src/common/featureFlags.ts [hunk 1]
```diff
@@ -13,6 +13,20 @@ export function isFeatureFlagEnabled(featureFlagName: string, defaultValue = fal
   }
 }
 
+/**
+ * Update feature flags related to GA.
+ */
+export function initializeGAFeatureFlags(): void {
+  if (isFeatureFlagEnabled(FeatureFlagName.GAPreview, false)) {
```
Comment (HuihuiWu-Microsoft) [inline packages/fx-core/src/common/featureFlags.ts:20]: What if user changed one of the feature flags after fxcore initialized?

File: packages/vscode-extension/package.json [hunk 2]
```diff
@@ -463,6 +463,11 @@
         "title": "%teamstoolkit.commands.addCapability.title%",
         "enablement": "!fx-extension.isSPFx && !fx-extension.isM365"
       },
+      {
+        "command": "fx-extension.addFeature",
+        "title": "Teams: Add Feature",
```
Comment (HuihuiWu-Microsoft) [inline packages/vscode-extension/package.json:468]: Move to package.nls.json?

File: packages/vscode-extension/src/handlers.ts [hunk 3]
```diff
@@ -624,6 +624,83 @@ export async function addCapabilityHandler(args?: any[]): Promise<Result<null, F
   return result;
 }
 
+export async function addFeatureHandler(args?: any[]): Promise<Result<null, FxError>> {
+  // TODO: move the question model to fx-core addFeature API
```
Comment (HuihuiWu-Microsoft) [inline packages/vscode-extension/src/handlers.ts:628]: Send telemetry here?

File: packages/vscode-extension/src/handlers.ts [hunk 4]
```diff
@@ -624,6 +624,83 @@ export async function addCapabilityHandler(args?: any[]): Promise<Result<null, F
...
+      },
+      {
+        id: "cicd",
+        label: `${localize("teamstoolkit.handlers.addFeature.cicd.label")}`,
+        detail: localize("teamstoolkit.handlers.addFeature.cicd.detail"),
+      },
+    ],
+  };
+  const stack = [addFeatureConfig];
+  while (stack.length > 0) {
```
Comment (HuihuiWu-Microsoft) [inline packages/vscode-extension/src/handlers.ts:674]: Why not ask the questions in fx-core and leverage existing question model(maybe move these questions to getQuestionsForAddFeature())? You can also benefit from question model for other platforms.

File: packages/fx-core/src/common/featureFlags.ts (hunk 1 above)
Comment (tecton) [reply packages/fx-core/src/common/featureFlags.ts:20]: If user changes configuration in VSC, they are instructed to reload window to take effect. This function is exported and called by VSC and CLI during activation.

File: packages/vscode-extension/package.json (hunk 2 above)
Comment (tecton) [reply packages/vscode-extension/package.json:468]: Thanks, will fix this.

File: packages/vscode-extension/src/handlers.ts (hunk 3 above)
Comment (tecton) [reply packages/vscode-extension/src/handlers.ts:628]: This function delegates call to other existing handlers and they have their own telemetry events. I'd like to add telemetry for "Add feature" once the API in core is ready.

File: packages/vscode-extension/src/handlers.ts (hunk 4 above)
Comment (tecton) [reply packages/vscode-extension/src/handlers.ts:674]: Yes, this logic will be moved to fx-core later. I will change to invoking user task after it's ready.

**Review by HuihuiWu-Microsoft — APPROVED** [review]

**Review by jayzhang — APPROVED** [review]

**Review by IvanJobs — APPROVED** [review]
lgtm for bot plugin

**Pull request merged**